### PR TITLE
feat(data): Two more person ships: Intola and Verdorve (closed because of a git error)

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Content (Person Ships)**

## Summary

Follow-up to #136 and #135.
Intola is a Scout with two Heavy Lasers as armament. I wondered about giving it a Quantum Keystone, but that's a little spoilery.
Verdorve's captain loves plasma weapons and engines, and the Splinter Verdorve flies uses nothing but plasma.

Also increases the person spawn rate by a somewhat extreme rate: twenty minutes on average to two and a half.

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
Oh, the Avgi are in!

File: data/persons.txt [hunk 1]
```diff
@@ -2437,3 +2437,102 @@ ship "Cicada" "Cicada (Sanujab)"
...
+			"Guess what. The Republic leader is an Alpha. I wish I could be teleported to Earth."
+			"Nunki once paid me tribute, but no more. Or was it Bloodsea?"
+			"Shoot 'em. Bang bang."
+			"Gatling Guns are ridiculously overpowered and expensive."
+			"Exploring the Confederated Councils of the Outer Rim's former territory is a nightmare. How do they have so many Leviathans?"
+			"You'll never guess where I got my name... or what it is."
+			"The Splinter is Megaparsec's best ship. Those Protectors are too slow, and Quicksilvers can't hold Plasma Turrets."
+			"Plasma, plasma. Overheat, overheat. Shoot, shoot."
+			"I wish I could join the Navy, but they won't let me. I refuse to give up this Splinter."
+			"I wish I could join the Syndicate, but they won't let me. For some reason, the corporate leaders have a predjuice against Southbound-sold weapons. Poor plasma."
```
Comment (TheGiraffe3) [inline data/persons.txt:2514]:
```suggestion
			"I wish I could join the Syndicate, but they won't let me. For some reason, the corporate leaders have a prejudice against Southbound-sold weapons. Poor plasma."
```

**Comment by Zitchas** [comment]
Ah.... There was a mishap with experimental. For some reason my combining of two branches on my own fork merged Azure's Avgi into experimental. I have reset it, so it's clean again, but your PR picked it up already. 

I'm sure the Avgi are going to be in eventually (probably sooner rather than later), but they're still being worked on, so the time is not yet. (and it would be squash merged, not commit-merged)

**Comment by TheGiraffe3** [comment]
I'll re-PR this without the extra commits.

**Comment by Zitchas** [comment]
Thanks, and sorry for the problem. I'm still not sure why that even happened. My guess is that somehow git created a symbolic link or something so pushing changes to the branch proceeded to push those on to experimental.

**Pull request closed without merging**